auditwheel/wheel_abi.py

```diff
@@ -112,7 +113,38 @@ def analyze_wheel_abi(wheel_fn: str):
     log.debug('external reference info')
     log.debug(json.dumps(external_refs, indent=4))
 
+    # get versioned symbols from external_refs
+    # since linux tag (priority 0) has no white-list, do not analyze it
+    external_ref_no_linux = dict(set([(v, k) for e in external_refs.values() if e['priority'] != 0 for k, v in e['libs'].items() if v is not None]))
```

Comment:
I pushed a new commit that I hope clarify things a bit. I'll squash it in the previous commit if that works for you otherwise, let's iterate a bit more before doing that.